2021-08-16 05:43:53,021 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2021-08-16 05:43:53,021 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Users\skyem\AppData\Local\Programs\Python\Python39\lib\site-packages\instabot\bot\bot.py
2021-08-16 05:43:54,330 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2021-08-16 05:43:54,489 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2021-08-16 05:43:54,489 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2021-08-16 05:43:54,489 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2021-08-16 05:43:54,489 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
